### ISSUE TITLE
Get this to work on Windows

### ISSUE THREAD

**Issue by ajvincent** [ajvincent/motherhen#23]
This is one I _can't_ fix, because I don't have a Windows operating system at hand.

**Comment by splatboydev** [comment]
I can take on the task.

**Comment by jthomp** [comment]
Perhaps you could try building/debugging against a Windows VM? https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/

**Comment by ajvincent** [comment]
@jthomp thanks for the suggestion. That might be what I end up doing... a few weeks from now.  

For personal reasons, I need to step away from active development on Motherhen at the moment.  I don't mind reviewing patches from others, but I can't do this myself right now.

I'd still very much like someone else to work through this in the meantime!

**Comment by inspector71** [comment]
Kudos for your substantial efforts

However, there's been the likes of something trying to support this obvious use case so many times and the real issue is a lack of resources. If Mozilla is not serious about investing in this kind of project, is it really ever going to be reliable enough to use? 

Are developers just going to suffer from a lack of true support for the idea much like days when Mozilla killed a lot of developer's efforts by destroying XUL based extensions?

See Songbird / Nightingale

**Comment by inspector71** [comment]
Perhaps one way to hopefully get this use case more supported would be to attract funding through the likes of Open Collective, Patreon?

How is the Electron project managed?

Who knows, Mozilla might even chip in themselves, supporting this use case, for once.

**Comment by ajvincent** [comment]
While I'm certainly willing to discuss these ideas, @inspector71 , let's take them off this issue, which is specifically for testing on Windows and fixing whatever bustages there are.

https://github.com/ajvincent/motherhen/discussions is probably a better home for these conversations.

**Comment by inspector71** [comment]
Absolutely, sorry, I was unsure if there was a better place to discuss.
